.github/workflows/cross-compiles.yml

```diff
@@ -47,72 +47,86 @@ jobs:
             target: no-threads 386 DJGPP,
             tests: none,
             ppa: jwt27/djgpp-toolchain
+            cflags:
           }, {
             arch: aarch64-linux-gnu,
             libs: libc6-dev-arm64-cross,
             target: linux-aarch64
+            cflags:
           }, {
             arch: alpha-linux-gnu,
             libs: libc6.1-dev-alpha-cross,
             target: linux-alpha-gcc
+            cflags:
           }, {
             arch: arm-linux-gnueabi,
             libs: libc6-dev-armel-cross,
             target: linux-armv4,
             tests: -test_includes -test_store -test_x509_store
+            cflags:
           }, {
             arch: arm-linux-gnueabihf,
             libs: libc6-dev-armhf-cross,
             target: linux-armv4,
             tests: -test_includes -test_store -test_x509_store
+            cflags:
           }, {
             arch: hppa-linux-gnu,
             libs: libc6-dev-hppa-cross,
             target: -static linux-generic32,
             fips: no,
             tests: -test_includes -test_store -test_x509_store
+            cflags:
           }, {
             arch: m68k-linux-gnu,
             libs: libc6-dev-m68k-cross,
             target: -static -m68040 linux-latomic -Wno-stringop-overflow,
             fips: no,
             tests: -test_includes -test_store -test_x509_store
+            cflags: -mno-xgot
```

Comment:
Does it mean we will need a new build target for m68k-linux-gnu that will automatically add this? Because this is just a workaround.

Reply:
No, I think its fine to just set this for the purposes of CI, and not include a new build target.  The only reason we need this is because m68k target we build for CI enables enough features that the binary overruns the default got size.  Other users of this platform may choose to disable features to shrink the binary, or do the same thing during their build as above